charts/common/tests/deployment_test.yaml

```diff
@@ -261,3 +261,20 @@ tests:
       - isNotEmpty:
           template: deployment.yaml
           path: spec.template.spec.containers[0].livenessProbe.exec.command
+  - it: must use terminationGracePeriodSeconds if gt than 30
+    set:
+      <<: *values
+      terminationGracePeriodSeconds: 60
+    asserts:
+      - equal:
+          path: spec.template.spec.terminationGracePeriodSeconds
+          value: 60
+  - it: skip use of terminationGracePeriodSeconds if missing or less to 31
```

Comment:
Is there a rationale behind this? Maybe I'm missing something :)

Reply:
changed to allow any value